src/components/allReviews/allReviewsCard.jsx

```diff
@@ -0,0 +1,9 @@
+export default function AllReviewsCard({ title, owner, designer, review_img_url, votes, category, comments, review_id }) {
+	return (<article className="review-card column" >
+		<h3>{title}</h3>
+		<img src={review_img_url} alt={designer} />
+		{/* <Link to={`/reviews/${item_id}`}>Click Here</Link> */}
```

Comment:
not sure what item_id is referring to in this Link, delete this bit of commented out code before merging this branch as this is more for the ticket to get single review(?) and will need to be set up in your router too